### ISSUE TITLE
Getting Deprecation Warning while using the fastjsonschema

### ISSUE THREAD

**Issue by rizbi-hassan** [horejsek/python-fastjsonschema#124]
Getting this warning message while running unit tests.
```
/venv/lib64/python3.7/site-packages/jsonschema/compat.py:6: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
    from collections import MutableMapping, Sequence  # noqa

-- Docs: https://docs.pytest.org/en/stable/warnings.html
```

I am using fastjsonschema version 2.15.0

The code where I am getting the error:
https://github.com/nasa/cumulus-orca/blob/develop/tasks/extract_filepaths_for_granule/test/unit_tests/test_extract_file_paths_for_granule.py

**Comment by horejsek** [comment]
Hey. Thanks for the report, but this is warning from jsonschema package. This package has dependency for this only for development, so should be fine for production code even if jsonschema does not fix it.
